public/js/vulx.localization.js

```diff
@@ -1,4 +1,4 @@
-const loadPath = `/locales/{{lng}}/{{ns}}.json`;
+const loadPath = `/locales/{{lng}}/{{ns}}.lang`;
```

Comment:
please do not change it from using a json to a lang as its not the correct format for i18next